addons/website_hr_recruitment/models/hr_recruitment.py

```diff
@@ -10,19 +10,30 @@
 class RecruitmentSource(models.Model):
     _inherit = 'hr.recruitment.source'
 
-    url = fields.Char(compute='_compute_url', string='Url Parameters')
+    url = fields.Char(related='link_tracker_id.short_url', string='Url Parameters')
+    link_tracker_id = fields.Many2one('link.tracker', compute='_compute_link_tracker_id', store=True)
+    click = fields.Integer(string='Clicks', related='link_tracker_id.count')
 
-    @api.depends('source_id', 'source_id.name', 'job_id')
-    def _compute_url(self):
-        base_url = self.env['ir.config_parameter'].sudo().get_param('web.base.url')
+    @api.depends('campaign_id', 'source_id', 'job_id.website_url')
+    def _compute_link_tracker_id(self):
+        web_base_url = self.env['ir.config_parameter'].sudo().get_param('web.base.url')
         for source in self:
-            source.url = urls.url_join(base_url, "%s?%s" % (source.job_id.website_url,
-                urls.url_encode({
-                    'utm_campaign': self.env.ref('hr_recruitment.utm_campaign_job').name,
-                    'utm_medium': self.env.ref('utm.utm_medium_website').name,
-                    'utm_source': source.source_id.name
-                })
-            ))
+            link_tracker = self.env['link.tracker'].create({
+                'url': str(web_base_url) + str(source.job_id.website_url),
```

Comment:
```suggestion
                'url': "%s%s" % (web_base_url + source.job_id.website_url)
```

You sure it's that one of the elements is never False ?